examples/client/test_object.c

```diff
@@ -64,7 +65,7 @@
  *  test |  1 |    R/W     |    No     |    Yes    | Integer | 0-255 |       |             |
  *  exec |  2 |     E      |    No     |    Yes    |         |       |       |             |
  *  dec  |  3 |    R/W     |    No     |    Yes    |  Float  |       |       |             |
- *
+ *  str  |  4 |    R/W     |    No     |    Yes    | String  |       |       |             |
```

Comment:
Leshan lists this as optional, not mandatory. See https://github.com/eclipse/leshan/blob/master/leshan-server-demo/src/main/resources/models/31024.xml.

Reply:
I have changed it resource id 5 to be non conflicting with the leshan object spec, but now I can't find it at all in the leshan project